src/test/java/rv/atqc/gttesting/archex/EditAvartarPage.java

```diff
@@ -0,0 +1,20 @@
+package rv.atqc.gttesting.archex;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class EditAvartarPage extends LogInPage  {
```

Comment:
Now EditAvatarPage extends from ProfilePage because after clicking "Confirm", ProfilePage begin contain all fields of EditAvatarPage. As for me this is "has-a" communication.